bin/workflows/pycbc_create_offline_search_workflow

```diff
@@ -213,6 +212,47 @@ statfiles = []
 statfiles += wf.setup_trigger_fitting(workflow, insps, hdfbank,
                                       final_veto_file, final_veto_name)
 
+# setup the detector combination coincs for the filtering jobs
+full_insps = insps
+ctags = [tag, 'full']
+bg_files = []
+final_bg_files = []
+ifoindeces = {}
+
+# Get the ifo precedent list for the multiifo coinc jobs
+ifo_precedence_list = workflow.cp.get_opt_tags('workflow-coincidence', 'timeslide-precedence', ctags)
+for ifo, file in zip(*insps.categorize_by_attr('ifo')):
+    ifoindeces[ifo] = ifo_precedence_list.index(ifo)
+
+for i in range(2,len(insps)+1):
+    combinations = itertools.combinations(ifoindeces.keys(),i)
```

Comment:
This line generates the detector combinations, at 2 it generates the 2 detector combination, if there are 3 trigger files, than it generates the 2 and 3 detector combination as well. Should there be 4, than it would create the 2, 3 and 4 combinations.